3.1/imports/README.md

```diff
@@ -72,3 +72,5 @@ class UsersController extends Controller
 ```
 
 :page_facing_up: Find the imported users in your database!
+
+:page_facing_up: By default the application will look for files in `/public`
```

Comment:
It actually doesn't. It looks whatever the location of your default disk is. If you changed your default disk, this can be different from Laravel's default, which should be `storage/app`

Reply:
Ah I see! I guess I'm pushing to have "where does it save by default" as part of the docs because every time I install this beautiful software I'm a bit confused at that part of the manual! Apologies I haven't done too many pull requests, what do you suggest, do I just close it for now?

Reply:
Perhaps change the message into something a bit more generic? Like referring to the filesystem config file to find the location?